src/pages/Trade/Swap/index.tsx

```diff
@@ -25,7 +25,7 @@ import useSlippageTolerance from "hooks/useSlippageTolerance";
 import { generateSwapMsg } from "utils/dezswap";
 import useBalance from "hooks/useBalance";
 import useFee from "hooks/useFee";
-import { XPLA_ADDRESS, XPLA_SYMBOL } from "constants/network";
+import { XPLA_ADDRESS } from "constants/network";
```

Comment:
It seems that XPLA_ADDRESS should not be left behind. The conditional statement below seems to need to be modified.